go/interpreter/lfvm/instructions_test.go

```diff
@@ -876,10 +900,10 @@ func TestExpansionCostOverflow(t *testing.T) {
 						}
 					}
 
-					test.op(&ctxt)
-
-					if ctxt.status != statusError {
-						t.Errorf("unexpected status, wanted not running, got %v, and gas of %v", ctxt.status, ctxt.gas)
+					err := test.op(&ctxt)
+					// FIXME: can we narrow the error? test says overflow, it manifests an out of gas error
```

Comment:
`memory.expandMemory` can produce two errors, size+offset overflow, or out of gas. it should be checked that the test parameters are actually producing an overflow and not just an out of gas error, or rename the test.

Reply:
The reason why I wrote a fixme is not to bloat this massive change even more. this test needs to be narrowed and that is a task in itself. to be documented in #751